video2dataset/dataloader/dataloader.py

```diff
@@ -35,8 +42,8 @@ def reassemble(x):
 def get_video_dataset(
     urls,
     batch_size,
-    shuffle=False,
-    repeat=False,
+    shuffle=0,
+    repeat=1,
```

Comment:
you only change these parameters here but nowhere else in the code. Is this intended?